src/sentry/static/sentry/app/components/group/times.tsx

```diff
@@ -24,6 +24,8 @@ const Times = ({lastSeen, firstSeen}: Props) => (
     <div
       css={css`
```

Comment:
Just a FYI because this isn't your problem per se - we discourage inline-styles on our components in favour of building styled-components at the bottom of the file (see `Container` component in this same file).